server/api/GET_helpers.py

```diff
@@ -0,0 +1,56 @@
+# all GET request helpers go in here
```

Comment:
You're going to have to run the following commands in your branch:

```bash
git merge main
```
since i refactored the rest of these, so your changes are one or two commits behind which will cause conflicts.